onnxruntime/featurizers_ops/cpu/pca_transformer.cc

```diff
@@ -51,6 +51,7 @@ struct PCATransformerImpl {
       output_matrix = val;
     };
     transformer.execute(input_matrix, callback);
+    transformer.flush(callback);
```

Comment:
>transformer.flush(callback) [](start = 4, length = 27)

Here, the matrix is output again and overwrites the previous matrix? I think we need comments everywhere explaining how this works (does not work)?